src/resources/js/tickets-registration-page.js

```diff
@@ -538,6 +540,25 @@ window.tribe.tickets.registration = {};
 			}
 		}
 
+		// Validation for Tribe Horizontal Date Picker
+		if ( $input.hasClass( 'tribe-horizontal-date-picker-value' ) ) {
+			const wrapper = $input.closest( '.tribe-horizontal-date-picker-container' );
+			const day = wrapper.find('.tribe-horizontal-date-picker-day');
+			const month = wrapper.find('.tribe-horizontal-date-picker-month');
+			const year = wrapper.find('.tribe-horizontal-date-picker-year');
+
+			[ day, month, year ].forEach( el => {
```

Comment:
I don't think we can use arrow functions here, as we probably still need to support IE11?

Reply:
Done. Thanks! https://github.com/moderntribe/event-tickets/pull/1656/commits/7dd47723256d678b95c5a04f368997c290025434